webhookURLValidator.go

```diff
@@ -115,11 +116,15 @@ func GoodAlternativeURLs(vs []ValidURLFunc) ValidatorFunc {
 	}
 }
 
-// HTTPSOnlyEndpoints creates a ValidURLFunc that considers a URL valid if the scheme
-// of the address is https.
-func HTTPSOnlyEndpoints() ValidURLFunc {
+// HTTPSOnlyEndpoints creates a ValidURLFunc that checks the scheme of the URL.
+// If t is true, then it will only allow URLs with "https" schemes.
+// If t is false, it will only allow URLs with "https" and "http" schemes.
```

Comment:
```suggestion
// GoodURLScheme creates a ValidURLFunc that checks the scheme of the URL.
// If httpsOnly is true, then it will only allow URLs with "https" schemes.
// If httpsOnly is false, it will only allow URLs with "https" and "http" schemes.
```